src/js/utils/formatting/dasherize.js

```diff
@@ -0,0 +1,5 @@
+import trim from 'js/utils/formatting/trim';
```

Comment:
we don't actually seem to be using this.. Should we keep it around?  Maybe

Reply:
Ah right, it's used by `slugify()`, but we removed that as well. I can remove this